project.json

```diff
@@ -1,9 +1,10 @@
 {
 	"version": "1.0.0-*",
-	"buildOptions": {
-		"emitEntryPoint": true,
-		"debugType": "portable"
-	},
+  "buildOptions": {
+    "emitEntryPoint": true,
+    "debugType": "portable",
+    "preserveCompilationContext": true
+  },
```

Comment:
Indentation is still not right for this file.  It may be be caused by Visual Studio editing the file on your behalf (for example when adding/removing NuGet packages.